src/concurrency/ocf_metadata_concurrency.c

```diff
@@ -215,10 +215,8 @@ int ocf_metadata_hash_try_lock(struct ocf_metadata_lock *metadata_lock,
 		ENV_BUG();
 	}
 
-	if (!result)
-		return -1;
 
-	return 0;
+	return result;
```

Comment:
This changes meaning of the result without update in places when this function is used. In result this commit breaks bisectability.

Reply:
actually this function is not referenced anywhere at this point. it starts being used in the next commit, also being changed to "status"